azurerm/internal/services/compute/virtual_machine_resource.go

```diff
@@ -621,6 +621,13 @@ func resourceArmVirtualMachine() *schema.Resource {
 			},
 
 			"tags": tags.Schema(),
+
+			"encryption_at_host": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: false,
+				Default:  false,
+			},
```

Comment:
The `azurerm_virtual_machine` resource is now in feature-frozen state, therefore could we revert the feature changes in this file?

Reply:
You can find more information about the feature-frozen here - https://www.terraform.io/docs/providers/azurerm/r/virtual_machine.html